packages/react-core/src/components/NumberInput/NumberInput.tsx

```diff
@@ -122,10 +132,15 @@ export const NumberInput: React.FunctionComponent<NumberInputProps> = ({
           </span>
         </Button>
         <input
-          className={css(styles.formControl)}
+          className={css(
+            styles.formControl,
+            validated === ValidatedOptions.success && formControlStyles.modifiers.success,
```

Comment:
@mcoker Should these `formControlStyles` be available in the number input.  If so, should we open a core issue?